docs/ocis/migration.md

```diff
@@ -10,38 +10,117 @@ geekdocFilePath: migration.md
 The migration happens in subsequent stages while the service is online. First all users need to migrate to the new architecture, then the global namespace needs to be introduced. Finally, the data on disk can be migrated user by user by switching the storage driver.
 
 {{< hint warning >}}
-@jfd: It might be easier to introduce the spaces api in oc10 and then migrate to ocis. We cannot migrate both at the same time, the architecture to ocis (which will change fileids) and introduce a global namespace (which requires stable fileids to let clients handle moves without redownloading). Either we implement arbitrary mounting of shares in ocis / reva or we make clients and oc10 spaces aware.
+@jfd: It might be easier to introduce the spaces api in oc10 and then migrate to oCIS. We cannot migrate both at the same time, the architecture to oCIS (which will change fileids) and introduce a global namespace (which requires stable fileids to let clients handle moves without redownloading). Either we implement arbitrary mounting of shares in oCIS / reva or we make clients and oc10 spaces aware.
 {{< /hint >}}
 
-## User Stories
-- As an admin, I need to avoid downtime.
-- As an admin, I want to migrate certain groups of users before others.
-- As a user, I need a seamless migration and not lose data by any chance.
-
 ## Migration Stages
 
-### Stage-0
+### Stage 0: _pre migration_
 Is the pre-migration stage when having a functional ownCloud 10 instance.
 
-### Stage-1
-Introduce OpenID Connect to ownCloud 10 server and clients.
+#### FAQ
+_Feel free to add your question as a PR to this document using the link at the top of this page!_ 
+
+### Stage 1: _introduce ownCloud Web_
+Install and introduce [ownCloud Web](https://github.com/owncloud/web/) and let users test it voluntarily to gain early feedback on the new UI.
+
+#### Steps
+Deploy web and enable switching to and from it.
+For more details see: [ownCloud 10 with ownCloud Web]({{< ref "deployment/owncloud10_with_oc_web.md" >}})
+_TODO @butonic is there documentation how to limit the web ui switch to an 'early adopters' group?_
+
+#### Validation
+Ensure switching back an forth between the classic ownCloud 10 web UI and ownCloud web works as at our https://demo.owncloud.com. 
+
+#### Rollback
+Should there be problems with ownCloud web at this point it can simply be removed from the menu and be undeployed. 
+
+#### Notes
+The ownCloud 10 demo instance uses OAuth to obtain a token for ownCloud web. In oCIS the token is provided by the OpenID Connect Identityd Provider, which may skip the consent step for trusted clients for a seamless login experience.
+
+#### FAQ
+_Feel free to add your question as a PR to this document using the link at the top of this page!_ 
+
+### Stage 2: _introduce OpenID Connect_
+
+Basic auth requires us to properly store and manage user credentials. Something we would rather like to delegate to a tool specifically built for that task.
```

Comment:
I wrote a blog post on this some time ago. Maybe you want to reference it https://owncloud.com/news/openid-connect-oidc-app/